Dockerfile

```
@@ -43,13 +43,23 @@ RUN
 # Append to /etc/profile for login shells
 RUN echo 'export PATH="/usr/local/go/bin:$PATH"' >> /etc/profile
 
-# Create .cargo directory
-RUN mkdir -p /root/.cargo
+# Create rust related directories in /usr/local
+RUN mkdir -p /usr/local/cargo
+RUN mkdir -p /usr/local/rustup
```

AFAICS, every RUN command creates another layer, maybe reduce it by
```suggestion
RUN mkdir -p /usr/local/cargo /usr/local/rustup
```